packages/cetlib/package.py

```diff
@@ -46,14 +46,17 @@ class Cetlib(CMakePackage):
     depends_on("cetlib-except")
     depends_on("hep-concurrency", when="@3.18.01:", type=("build", "test"))
     depends_on("hep-concurrency", when="@:3.18.00")
-    depends_on("openssl")
+    with when("@3.14.00:"):
+        if sys.platform != "darwin":
+            depends_on("openssl")
+    depends_on("openssl", when="@:3.13")
     depends_on("perl", type=("build", "run"))
     depends_on("sqlite@3.8.2:")
-    depends_on("catch2@3:", when="@3.17:", type=("build", "test"))
-    depends_on("catch2@2.3.0:", when="@:3.16.99", type=("build", "test"))
-    depends_on("catch2", type=("build", "test"))
+    depends_on("catch2@3.3.0:", when="@3.17:", type=("build", "test"))
+    depends_on("catch2@2.3.0:2", when="@:3.16", type=("build", "test"))
     depends_on("cetmodules", type="build")
     conflicts("cetmodules@:3.21.00", when="catch2@3:")
+    depends_on("tbb")
```

Comment:
Is `tbb` not just a `build` or `test` dependency?

Reply:
It shouldn't be any kind of dependency, but it was added for a reason, and now I'm very unsure what that reasoning was. Checking now...

Reply:
Lots of these dependencies were added due to CMake being unable to find things / header files not being able to find other header files, and probably should have been build dependencies. (although if you need the header, you may really need the library, so...)